src/physics/cam/cam_diagnostics.F90

```diff
@@ -683,14 +747,6 @@ subroutine diag_init_moist(pbuf2d)
       call add_default ('CUSH&IC    ',0, 'I')
       call add_default ('KVH&IC     ',0, 'I')
       call add_default ('KVM&IC     ',0, 'I')
-      !Add these variables to the i.c. for spreads. DA radiances, RTTOV needs
```

Comment:
Please add these lines to your version. These are needed by the data assimilation group.